llama-index-packs/llama-index-packs-searchain/README.md

```diff
@@ -0,0 +1,6 @@
+# LlamaIndex Packs Integration: Searchain
+This LlamaPack implements short form the [SearChain paper by Xu et al..](https://arxiv.org/abs/2304.14732)
+
+You can see its use case in the examples folder.
```

Comment:
We can paste the example from the notebook here tbh (and we should also include an example of installing and/or downloading the pack)